tests/tensorflow/r2.12/common.libsonnet

```diff
@@ -23,9 +23,62 @@ local mixins = import 'templates/mixins.libsonnet';
 
     frameworkPrefix: 'tf-r2.12.0',
     tpuSettings+: {
-      softwareVersion: '2.12.0',
+      softwareVersion: '2.11.0',
```

Comment:
This can now be 2.12.0. Can you manually trigger a build and check if its working properly?

Reply:
Setting this to 2.12.0 will still result in an error because the debian image is not in prod yet.